File: langhuan-service/src/main/java/com/langhuan/model/pojo/ChatRestOption.java
package com.langhuan.model.pojo;

import lombok.Data;

import java.util.List;

@Data
public class ChatRestOption {
    String chatId;
    String prompt;
    // 对应userMessage
    String question;
    Boolean isRag;
    String ragGroupId;
    Boolean isReRank;
    Boolean isFunction;
    String modelName;
    List<String> imageunderstanding;
}

File: langhuan-server/src/main/kotlin/com/langhuan/model/pojo/ChatRestOption.kt
package com.langhuan.model.pojo

import lombok.Data

@Data
data class ChatRestOption(
    var chatId: String? = null,
    var prompt: String? = null,
    // 对应userMessage
    var userMessage: String? = null,
    var ragGroupId: String? = null,
    var isReRank: Boolean? = null,
    var isFunction: Boolean? = null,
    var modelName: String? = null
)
